backend/routing/routing.go

```diff
@@ -0,0 +1,36 @@
+package routing
```

Comment:
There's also a slew of route helpers in `backend/alerts/routing.go`
Ideally those functions would go here. However, because the slack logic is in `model.go` (and not `backend/alerts`), we get a circular dependency (because those routes reference model code).

cc @SpennyNDaJets - when we take on refactoring slack alerts, we should move it to `backend/alerts/slack` and we can make the logic in `backend/alerts/routing.go` more useful to other parts of the codebase.